data_taking_certification/views.py

```diff
@@ -3,58 +3,45 @@
 from data_taking_objects.models import Run, Lumisection
 from data_taking_certification.models import RunCertification, LumisectionCertification
 
-# from histograms.models import RunHistogram
-
-import pandas as pd
-
 
 # Create your views here.
 def run_certification_view(request):
 
     error_message = None
-    df = None
-
-    # TODO The following lines should be done on the DB side
-    certification_df = pd.DataFrame(RunCertification.objects.all().values())
-    runs_df = pd.DataFrame(Run.objects.all().values())
-
-    if certification_df.shape[0] > 0:
-        df = runs_df.merge(certification_df, left_on="id", right_on="run_id")[
-            [
-                "run_number",
-                "rr_frac_pixel_good",
-                "rr_frac_strip_good",
-                "rr_frac_tracking_good",
-                "rr_frac_ecal_good",
-            ]
-        ]
-        print(df.head())
+
+    run_certifications = RunCertification.objects.all()
+    n_certifications = len(run_certifications)
+
+    if n_certifications > 0:
+        print(f"{n_certifications} certifications are being loaded")
```

Comment:
Also avoid prints and used a `logger` wherever possible!

Reply:
And done.